pkg/deploy/generator/templates_rp.go

```diff
@@ -84,6 +87,8 @@ func (g *generator) rpTemplate() *arm.Template {
 			"armClientId",
 			"billingServicePrincipalId",
 			"billingE2EStorageAccountId",
+			"extraCosmosDBIPs",
```

Comment:
Is there any reason to add this back? 
https://github.com/Azure/ARO-RP/pull/1561#issue-677937338
```suggestion
			
```

Reply:
Fixed - most likely a merge change introduced on rebase.  Thank you!